From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Martijn Muijsers <martijnmuijsers@live.nl>
Date: Thu, 1 Dec 2022 17:26:18 +0100
Subject: [PATCH] Use timin.gs by default

Removed since 1.21

License: GPL-3.0 (https://www.gnu.org/licenses/gpl-3.0.html)
Gale - https://galemc.org

This patch is based on the following patch:
"Optimize default values for configs"
By: etil2jz <81570777+etil2jz@users.noreply.github.com>
As part of: Mirai (https://github.com/etil2jz/Mirai)
Licensed under: GPL-3.0 (https://www.gnu.org/licenses/gpl-3.0.html)

diff --git a/src/main/java/io/papermc/paper/configuration/GlobalConfiguration.java b/src/main/java/io/papermc/paper/configuration/GlobalConfiguration.java
index 379452394679c466a09441d4c12882d0d00d9ef3..b021721efaab5242b7d174a9c859fe06a5ca6054 100644
--- a/src/main/java/io/papermc/paper/configuration/GlobalConfiguration.java
+++ b/src/main/java/io/papermc/paper/configuration/GlobalConfiguration.java
@@ -98,7 +98,7 @@ public class GlobalConfiguration extends ConfigurationPart {
     public class Timings extends ConfigurationPart {
         public boolean enabled = false;
         public boolean verbose = true;
-        public String url = "https://timings.aikar.co/";
+        public String url = "https://timin.gs/"; // Gale - use timin.gs by default
         public boolean serverNamePrivacy = false;
         public List<String> hiddenConfigEntries = List.of(
             "database",
